docs/man/xrdp.ini.5.in

```diff
@@ -25,6 +25,7 @@ The options to be specified in the \fB[Globals]\fR section are the following:
 
 .TP
 \fBaddress\fP=\fIip address\fP
+No longer supported. Specify address and port in the port parameter.
```

Comment:
Can you remove the reference to address entirely? As I alluded to in #1610, if we leave it in it will never get removed.

Reply:
I agree. BTW, it might be true that `address` is not working now however `address` is used in the code. We need to cleanup the code, too.

Reply:
Done, I didn't understand what you meant.

Reply:
`address` is not ignored in the code.